pennylane_lightning/lightning_kokkos/_measurements.py

```diff
@@ -0,0 +1,481 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Class implementation for state vector measurements.
+"""
+
+# pylint: disable=import-error, no-name-in-module, ungrouped-imports
+try:
+    from pennylane_lightning.lightning_kokkos_ops import MeasurementsC64, MeasurementsC128
+except ImportError:
+    pass  # HELP: Should be a complaint when kokkos_ops module is not available.
```

Comment:
Why here have "pass" instead of a warning message?

Reply:
Is this required for building docs? If not, this should raise an error as LK doesn't fallback to DQ anymore (the same for LQ and LGPU).